contracts/contracts/sbtc-deposit.clar

```diff
@@ -14,11 +14,9 @@
 ;;
 
 ;; public functions
-
 ;; Accept a new deposit request
 ;; Note that this function can only be called by the current
 ;; bootstrap signer set address - it cannot be called by users directly.
-;; 
 ;; This function handles the validation & minting of sBTC, it then calls
 ;; into the sbtc-registry contract to update the state of the protocol
 (define-public (complete-deposit-wrapper (txid (buff 32)) (vout-index uint) (amount uint) (recipient principal))
```

Comment:
I think it's actually better to revert this change. When you run `pnpm clarigen docs`, which generates markdown docs, this will make the doc comment for this function be

```md
public functions Accept a new deposit ...
```

Same with the "newline" - when converted to markdown it'll start a new paragraph, which I think is helpful.

Reply:
Ehhh, I personally don't like the new-line / extra spacing. Nothing to stand my ground but it looks odd to have separated multiple line comments for a single function.